Traceback (most recent call last):
  File "/Users/stevenqueen/Projects/Python/autodownload/autodownload/checkmail.py", line 14, in <module>
    checkmail(sys.argv[1])
IndexError: list index out of range
